src/StoreApi/Routes/AbstractCartRoute.php

```diff
@@ -102,6 +116,29 @@ protected function requires_nonce( \WP_REST_Request $request ) {
 		return 'GET' !== $request->get_method();
 	}
 
+	/**
+	 * Triggered after an update to cart data. Re-calculates totals and updates draft orders (if they already exist) to
+	 * keep all data in sync.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 */
+	protected function cart_updated( \WP_REST_Request $request ) {
```

Comment:
Not saying it's wrong because I'm not sure - but why is this in the past tense?

Reply:
In the docblock I wrote "Triggered after an update to cart data", so I guess because the cart has already updated by this point.